dev/dags/dbt/jaffle_shop/.user.yml

```diff
@@ -0,0 +1 @@
+id: 3ff83655-3dec-45dc-b611-05c39b864ceb
```

Comment:
@abhi12mohan please, could you remove this file, it shouldn't be committed - I'll do a separate PR to add it to `.gitignore`